### PR TITLE
[#2318][#2319][#2320][#2321][#1904][#1902][#1900][#1874] Improvement(all): Make repeated string a constant

### PR DESCRIPTION
**Pull request by caican00**
### What changes were proposed in this pull request?

Make repeated error string a constant

### Why are the changes needed?

There is only one place to change when optimizing the prompt.

Fix: #2318 
Fix: #2319 
Fix: #2320 
Fix: #2321  
Fix: [#1904](https://github.com/datastrato/gravitino/issues/1904)
Fix: [#1902](https://github.com/datastrato/gravitino/issues/1902)
Fix: [#1900](https://github.com/datastrato/gravitino/issues/1900)
Fix: [#1874](https://github.com/datastrato/gravitino/issues/1874)

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Locally test.

### CODE REVIEW

**Comment by jerryshao** [comment]
@caican00 Thanks for the PR. Would you please fix all the related issues as possible as you can, not just a single file.

**Comment by caican00** [comment]
> @caican00 Thanks for the PR. Would you please fix all the related issues as possible as you can, not just a single file.
@jerryshao Is it recommended to fix all the related issues in one pr?

**Comment by jerryshao** [comment]
Yes, for such trivial issues, I would suggest to fix all the similar things in one PR as possible as you can.

**Comment by caican00** [comment]
> Yes, for such trivial issues, I would suggest to fix all the similar things in one PR as possible as you can.

Get.

**Comment by caican00** [comment]
@jerryshao @justinmclean  I put all the fixes in this pr and could you help to review it? Thank you.

**Comment by jerryshao** [comment]
Did you fix all the issues here and in other places besides these issues?

image

**Comment by jerryshao** [comment]
I think you have to update the PR title and description like other PRs to add the issues you fixed.

File: catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogOperations.java [hunk 1]
```diff
@@ -53,6 +53,9 @@
 
 public class HadoopCatalogOperations implements CatalogOperations, SupportsSchemas, FilesetCatalog {
 
+  private static final String SCHEMA_DOES_NOT_EXIST = "Schema %s does not exist";
```
Comment (jerryshao) [inline catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogOperations.java:56]: Can you please rename all the string to "XXX_MSG", add "_MSG" suffix to all these string variable names.

**Comment by caican00** [comment]
> Did you fix all the issues here and in other places besides these issues?
> 
> image

Yes.

File: catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogOperations.java (hunk 1 above)
Comment (caican00) [reply catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogOperations.java]: @jerryshao Thank you for reviewing. I have updated it.

**Comment by caican00** [comment]
> Did you fix all the issues here and in other places besides these issues?
> 
> image

yes.

**Comment by jerryshao** [comment]
LGTM. Thanks @caican00 for your fix.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**